6-register/1-register.md

```diff
@@ -0,0 +1,67 @@
+# Register a Rococo Parachain
+
+Rococo is Parity's official test network for Cumulus-based parachains. The purpose of this document is to guide Cumulus
+parachain developers through the steps needed to register their parachain with the Rococo test network.
+
+## Write Your Parachain
+
+Follow the steps in the previous chapter to develop your parachain. In particular, refer to the
+[Parachain Template Overview](../5-develop/1-template-overview.md) and
+[Template Pallet](../5-develop/3-template-pallet.md) sections.
+
+## Test Your Parachain
+
+Follow the steps in the previous section to test your parachain. In particular, refer to the
+[Sending Messages](../5-develop/4-sending-messages.md) and [Receiving Messages](../5-develop/5-receiving-messages.md)
+sections.
```

Comment:
I'm fine with going in exactly as it is written here. But I kind of prefer an approach that says "we expect you've followed this entire workshop and understood the materials before trying to use the public network"